css/style.css

```diff
@@ -0,0 +1,22 @@
+body {
+	text-align: center;
+	margin-left: auto;
+	margin-right: auto;
+	background-image: url("pics/background.png");
```

Comment:
Your background isn't showing up due to an incorrect path. We need to specify a path relative to the location of the current file, like so:
```
	background-image: url("../pics/background.png");
```